### ISSUE TITLE
Please update Documentation

### ISSUE THREAD

**Issue by stepkun** [mbocquet/keyboard-configuration#1]
Hi,

in your documentation you name the role with a dash: keyboard-configuration .
It sould be named with an underscore: keyboard_configuration .
Best would be to use the fully quallyfied name mbocquet.keyboard_configuration  .

Kind regards
stepkun

**Comment by mbocquet** [comment]
Hi.

You're right !

The role have been created before I met ansible-lint. I named it with '-' as in the OS package named keyboard-configuration.

For the use of the fully qualified name, It seems you're pointing the ansible-galaxy method to install a role.
As I don't use ansible-galaxy nor requirements.yml to download or update roles myself, I don't mention it in my files. I prefer to use the git submodule feature, directly pointing to a git repository avoiding ansible-galaxy which is tied to github.

Like this :
`git submodule add https://url.of.git/keyboard-configuration.git roles/whatever_you_choose_to_name_this_role_in_your_ansible_tree`

Without my username etc.

Indeed, I just checked the output of those commands :

`ansible-galaxy search keyboard-configuration`

```
Found 1 roles matching your search:

 Name                            Description
 ----                            -----------
 mbocquet.keyboard_configuration Ansible role to configure keyboard layout
```

ansible-galaxy converts the role-name into role_name in its search.

The `ansible-galaxy install` command creates a folder `mbocquet.keyboard_configuration`. Its up to your requirements.yml to place the role in an anonymous folder like `roles/keyboard_configuration`.

Anyway, I will soon rename this role to '_' style as its confusing with the ansible-galaxy behaviour.

Kind regards.
Mathias.
